Src/PerformanceCollector/Filtering/Implementation/Service contract/CollectionConfigurationInfo.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Filtering
+{
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Represents the entire collection configuration for this SDK.
```

Comment:
Done.

DTO that represents the collection configuration - a customizable description of performance counters, metrics, and full telemetry documents to be collected by the SDK. Processed and encapsulated by <see cref="CollectionConfiguration"/> at the time of actual collection.